webpack/assets/javascripts/react_app/components/StatisticsChartsList/index.js

```diff
@@ -1,71 +1,32 @@
 import React from 'react';
-import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
 
-import ChartBox from '../ChartBox';
-import * as StatisticsChartActions from '../../redux/actions/statistics';
-import { STATUS } from '../../constants';
+import ConnectedChartBox from '../ChartBox';
 import './StatisticsChartsListStyles.scss';
 import { translate as __ } from '../../common/I18n';
-import { noop } from '../../common/helpers';
 
-const getStatusFromChart = chart => {
-  if (chart.data) {
-    return STATUS.RESOLVED;
-  }
-  if (chart.error) {
-    return STATUS.ERROR;
-  }
-  return STATUS.PENDING;
+const StatisticsChartsList = ({ data }) => {
+  const chartBoxes = Object.values(data).map(chart => (
+    <ConnectedChartBox
+      key={chart.id}
+      type="donut"
+      chart={chart}
+      noDataMsg={__('No data available')}
+      tip={__('Expand the chart')}
+    />
+  ));
+
+  return (
+    <div className="statistics-charts-list-root cards-pf">{chartBoxes}</div>
+  );
```

Comment:
imo we don't really need this component,
instead of `Statistics` ---> `StatisticsChartsList` ---> `ConnectedChartBox`
we could render `ConnectedChartBox` directly from `Statistics`: 

```js
const Statistics = ({ statisticsMeta }) => {
  if (!statisticsMeta) {
    return <p>{__('No Charts Loaded')}</p>;
  }

  const chartBoxes = Object.values(statisticsMeta).map(chart => (
    <ConnectedChartBox
      key={chart.id}
      type="donut"
      chart={chart}
      noDataMsg={__('No data available')}
      tip={__('Expand the chart')}
    />
  ));

  return (
    <div className="statistics-charts-list-root cards-pf">{chartBoxes}</div>
  );
};
```

Reply:
or even if you want to take this refactor higher - move this code into `StatisticsPage.js`
this could be in a separate PR too since more refactoring should happen there.

Reply:
I think its good to keep the list component and the page component separated, and maybe in a different PR refactor the `Statistics` and   `StatisticsPage`

Reply:
imo this is not a component which will be used elsewhere,
so we could reduce the files and tests for unneeded files such as `StatisticsChartsList`,
I am OK with moving the refactor of `StatisticsPage` into a separate PR because it's more complicated.

Reply:
created a redmine for it: https://projects.theforeman.org/issues/28897